topics/dev/tutorials/writing_tests/api2.md

```diff
@@ -0,0 +1,31 @@
+
+> ### {% icon solution %} Solution
+> 
+> API test for verifying the list of datatypes.
+> 
+> ```python
+> from ._framework import ApiTestCase
+> 
+> 
+> NUMBER_OF_DATATYPES = 446
+> 
```

Comment:
This test depends on a very specific Galaxy configuration and shouldn't be an API test IMO. Perhaps an integration test... but even then I think the other assertions are better and more robust. Can we just drop this number of datatypes checking. Maybe verify it is a list?

Reply:
I was just looking for a simple list of "things" - I'll just use another example.